### ISSUE TITLE
Question about Button component

### ISSUE THREAD

**Issue by AgtLucas** [bloodyowl/reReactPrimitives#2]
Hey @bloodyowl, why are you using a `div` for the Button component?

Thanks!

**Comment by bloodyowl** [comment]
it's mostly for styling purposes. there are quite a few issues with flexbox and the `button`. I'll add some missing accessibility props in the next few days.

**Comment by AgtLucas** [comment]
@bloodyowl Got it! I thought was for something else.

I'm curious now, which issues did you faced with `flexbox` and `button`?

**Comment by bloodyowl** [comment]
https://github.com/philipwalton/flexbugs#9-some-html-elements-cant-be-flex-containers

**Comment by AgtLucas** [comment]
@bloodyowl I see! I'll try to help if you don't mind :)

**Comment by bloodyowl** [comment]
Accessibility props are on!
